server/index.ts

```diff
@@ -72,27 +72,15 @@ app.use(
 	}),
 )
 
-async function getRequestHandlerOptions(
-	build: ServerBuild,
-): Promise<Parameters<typeof createRequestHandler>[0]> {
+function getRequestHandler(build: ServerBuild): RequestHandler {
 	function getLoadContext(_: any, res: any) {
 		return { cspNonce: res.locals.cspNonce }
 	}
-	return { build, mode: MODE, getLoadContext }
+	return createRequestHandler({ build, mode: MODE, getLoadContext })
 }
 
-app.all(
-	'*',
-	process.env.NODE_ENV === 'development'
-		? async (req, res, next) => {
-				return createRequestHandler(await getRequestHandlerOptions(devBuild))(
```

Comment:
This change won't work because we need to make certain every request uses the latest version of the `devBuild` variable. With the changes here it will only ever use the first version.